Source/WebCore/css/CSSGridLineValue.cpp

```diff
@@ -63,22 +63,17 @@ Ref<CSSGridLineValue> CSSGridLineValue::create(RefPtr<CSSPrimitiveValue>&& spanV
 
 bool CSSGridLineValue::equals(const CSSGridLineValue& other) const
 {
-    if (m_spanValue) {
-        if (!other.m_spanValue || !m_spanValue->equals(*other.m_spanValue))
+    auto equals = [](CSSPrimitiveValue* value, CSSPrimitiveValue* otherValue) {
+        if ((!value && otherValue) || (value && !otherValue))
             return false;
-    } else if (other.m_spanValue)
-        return false;
+        return (!value && !otherValue) || value->equals(*otherValue);
+    };
 
-    if (m_numericValue) {
-        if (!other.m_numericValue || !m_numericValue->equals(*other.m_numericValue))
-            return false;
-    } else if (other.m_numericValue)
+    if (!equals(protectedSpanValue().get(), other.protectedSpanValue().get()))
```

Comment:
Does it compile without `.get()` ? I would drop if them if so

Reply:
It wouldn't. a `Ref<T>` can implicitly get converted to a `T&` but a RefPtr<T>` cannot be implicitly converted to a `T*` with our current API.